RecoLocalMuon/GEMSegment/plugins/ME0SegmentBuilder.cc

```diff
@@ -47,7 +47,9 @@ void ME0SegmentBuilder::build(const ME0RecHitCollection* recHits, ME0SegmentColl
     // save current ME0RecHit in vector associated to the reference id
```

Comment:
the comment above says "have to mask also the rolls .. to recover segments that are at the border of a roll"
shouldn't this be done here?